lib/Dialect/TfheRustBool/IR/TfheRustBoolOps.td

```diff
@@ -49,6 +51,18 @@ def TfheRustBool_NorOp  : TfheRustBool_BinaryGateOp<"nor">  { let summary = "Log
 def TfheRustBool_XorOp : TfheRustBool_BinaryGateOp<"xor"> { let summary = "Logical XOR of two TFHE-rs Bool ciphertexts."; }
 def TfheRustBool_XnorOp : TfheRustBool_BinaryGateOp<"xnor"> { let summary = "Logical XNOR of two TFHE-rs Bool ciphertexts."; }
 
+def TfheRustBool_PackedOp : TfheRustBool_Op<"gates_packed", [
+    Pure]> {
+  let arguments = (ins
+    TfheRustBool_ServerKey:$serverKey,
+    TfheRustBoolGate_Attr:$gates,
```

Comment:
Tried to implement it that way and got an error: 
`lib/Dialect/TfheRustBool/IR/TfheRustBoolOps.td:54:5: error: unexpected def type; only defs deriving from TypeConstraint or Attr or Property are allowed
def TfheRustBool_PackedOp : TfheRustBool_Op<"gates_packed", [
    ^`

Reply:
I changed the attributes to use an Enum definition. Those enums still needed to be wrapped in an Attribute before it can be used as an input to the Operations